### ISSUE TITLE
Explain probability semantics and composition limits for Jev/GLiClass decision outputs

### ISSUE THREAD

**Issue by microprediction** [microprediction/latentreasoning#121]
## Scope

Companion to #120. This is a proposed conceptual addition for the new bibliography entries, with a short explanation on `prediction.html` / `routing.html`; it is not an allegation that the current site already makes these Jev-specific claims.

The important question is what the returned numbers mean, and what remains valid when code composes them.

## Keep these quantities separate

| Quantity | Meaning to document | What does not follow automatically |
|---|---|---|
| Schema-valid output | The answer belongs to the permitted type/options | The selected answer is factually correct |
| Categorical probabilities | Distribution over supplied alternatives for one selection task | Each alternative is an independently applicable label |
| Multi-label scores / probabilities | Separate scores for potentially overlapping labels | They sum to one, or specify a joint distribution |
| Distribution concentration / confidence statistic | A summary of the returned distribution | A numerically equal probability of correctness |
| Ordinal score | Position on a supplied rubric | A probability of an event or a ratio-scale measurement |
| Forecast | Probability for a specified future outcome | Calibration transfers from a classification benchmark |

### Concrete Jev and GLiClass anchors

- Jev's [confidence documentation](https://docs.typesafe.ai/confidence) describes `confidence` as a statistic derived from the Choice/Score distribution. Its [Choice example](https://docs.typesafe.ai/primitives/choice) returns probability **0.61** for the winning department and confidence **0.42**. Therefore the confidence field must not simply be read as “42% probability the selected answer is correct”; its interpretation and operating threshold require validation.
- Jev exposes [Noul](https://docs.typesafe.ai/primitives/noul) for binary propositions and [Score](https://docs.typesafe.ai/primitives/score) for rubric-based ordinal judgements. A normalized severity score is still not a forecast probability.
- GLiClass's [pipeline implementation](https://github.com/Knowledgator/GLiClass/blob/main/gliclass/pipeline.py) uses softmax for single-label mode and sigmoid for multi-label mode. Normalization/activation choice is not evidence of calibration or statistical independence.

## Parallel evaluation is not probabilistic independence

The [Jev introduction](https://docs.typesafe.ai/introduction) describes questions evaluated separately against shared state. That is an execution/interface property. It does not imply independence of the underlying events or errors.

For example, even if two numbers were exact conditional marginals,

```text
P(A | x) = 0.6
P(B | x) = 0.6
```

the sharp bounds are

```text
0.2 <= P(A and B | x) <= 0.6.
```

The product 0.36 requires conditional independence; it does not follow from parallel calls. Marginal calibration alone is weaker still. A legitimate deterministic policy can threshold both scores without asserting any probability for their conjunction. If the application needs that probability, query the conjunction, estimate the dependence, or retain bounds—and validate the resulting estimate separately.

Similarly, joint text/label contextualization in an encoder does not turn its vector of marginal outputs into a joint probability model.

## Calibration and oversight

Add [Guo, Pleiss, Sun & Weinberger (2017), On Calibration of Modern Neural Networks](https://proceedings.mlr.press/v70/guo17a.html) as an antecedent and explicitly distinguish calibration from discrimination and sharpness. For a binary forecast, calibration means `E[Y | p] = p` on the relevant population; a constant base-rate forecast can satisfy it while being unhelpful for selecting cases.

Recommend held-out Brier/log loss, reliability plots, and risk versus coverage for abstention, including domain/time shifts and changed label sets. Fit any recalibration and thresholds on validation data only. Do not treat a proper-score training objective or the name “RLCD” as a finite-sample or deployment calibration guarantee.

For monitorability, a typed decision log exposes the chosen action and its inputs, not necessarily the mechanism that produced it. Removing generated prose changes the available monitoring channel; it neither proves opacity nor supplies causal interpretability.

## Acceptance criteria

- Add the semantics table or an equivalent concise explanation beside the new family.
- Include the confidence-versus-probability example and the conjunction counterexample.
- Cross-reference existing proper-scoring and calibration discussion without conflating Jev's RLCD with RLCR.
- Avoid “cannot hallucinate” as a synonym for schema validity, and avoid unvalidated confidence thresholds presented as correctness guarantees.

**Comment by microprediction** [comment]
Folded into a short section on the prediction page in c070438 rather than the full table. It covers softmax versus independent sigmoid outputs, Jev's own example where confidence (0.42) differs from the chosen option's probability (0.61), and the 0.2 to 0.6 bound on a conjunction of two 0.6 marginals. It also notes that Guo et al. assume validation data matches the test distribution, which a training objective named for calibration can't guarantee after a shift. The ordinal-score and forecast rows didn't add enough here to justify a table.
